Project/Source/Components/UI/ComponentCanvasRenderer.cpp

```diff
@@ -23,12 +23,16 @@ void ComponentCanvasRenderer::Render(const GameObject* gameObject) const {
 		//IF OTHER COMPONENTS THAT RENDER IN UI ARE IMPLEMENTED, THEY MUST HAVE THEIR DRAW METHODS CALLED HERE
 		ComponentImage* componentImage = gameObject->GetComponent<ComponentImage>();
 		if (componentImage != nullptr) {
-			componentImage->Draw(transform2D);
+			if (componentImage->IsActive()) {
+				componentImage->Draw(transform2D);
+			}
```

Comment:
This IsActive should not be necessary. Inactive components should never have their Render() method called.

Reply:
True